kafka-rest/src/main/java/io/confluent/kafkarest/entities/BrokerConfig.java

```diff
@@ -0,0 +1,107 @@
+package io.confluent.kafkarest.entities;
+
+import java.util.Objects;
+import java.util.StringJoiner;
+import javax.annotation.Nullable;
+
+/**
+ * A Kafka Broker Config
+ */
+public class BrokerConfig {
+
+  private final String clusterId;
+
+  private final String brokerId;
```

Comment:
`brokerId` is an `int`, is it not?

Reply:
I thought of using string since ConfigResource accepts String, so it won't need additional conversion. Have converted to int now.